src/ensembl/src/content/app/entity-viewer/gene-view/components/gene-overview-image/GeneOverviewImage.scss

```diff
@@ -0,0 +1,60 @@
+@import 'src/styles/common';
+
+.container {
+  color: white;
+  display: grid;
+  grid-template-columns: 210px 18px 695px 27px 180px;
+  grid-template-rows: repeat(3, auto); // FIXME: unnecessary?
```

Comment:
Is this `FIXME` for now or for future reference?

Reply:
✔️